phpBB/styles/prosilver/theme/bidi.css

```diff
@@ -596,7 +604,11 @@ li.breadcrumbs span:first-child > a {
 */
 
 .rtl .caret {
-	float: right;
+	left: -6px;
+	right: inherit;
+	border-right: 1px solid;
+	border-right-color: inherit;
+	border-left: none;
 }
 
```

Comment:
pls lint your code the order of these properties is incorrect. https://stylelint.io/

should be 
```
.rtl .caret {
	border-right: 1px;
	border-right-color: inherit;
	border-left: none;
	right: inherit;
	left: -6px;
}
```